examples/seismic/model.py

```diff
@@ -50,61 +50,72 @@ class Model(object):
     :param m: The square slowness of the wave
     :param damp: The damping field for absorbing boundarycondition
     """
-    def __init__(self, origin, spacing, vp, nbpml=0, dtype=np.float32,
-                 rho=None, epsilon=None, delta=None, theta=None, phi=None):
+    def __init__(self, origin, spacing, shape, vp, nbpml=0, dtype=np.float32,
```

Comment:
This is necessary if the field is constant as only a vlaue is passed and `vp.shape = ()`